Spec/Auth.swift

```diff
@@ -1748,21 +1747,18 @@ class Auth : QuickSpec {
                     }
 
                     waitUntil(timeout: testTimeout) { done in
-                        delay(tokenParams.ttl + 1.0) {
-                            rest.auth.authorise(nil, options: nil) { tokenDetails, error in
-                                expect(error).to(beNil())
-                                guard let tokenDetails = tokenDetails else {
-                                    XCTFail("TokenDetails is nil"); done(); return
-                                }
-                                expect(tokenDetails.clientId).to(equal(ExpectedTokenParams.clientId))
-                                expect(tokenDetails.issued!.dateByAddingTimeInterval(ExpectedTokenParams.ttl)).to(beCloseTo(tokenDetails.expires))
-                                expect(tokenDetails.capability).to(equal(ExpectedTokenParams.capability))
-                                done()
+                        rest.auth.authorise(nil, options: nil) { tokenDetails, error in
+                            expect(error).to(beNil())
+                            guard let tokenDetails = tokenDetails else {
+                                XCTFail("TokenDetails is nil"); done(); return
                             }
+                            expect(tokenDetails.clientId).to(equal(ExpectedTokenParams.clientId))
+                            expect(tokenDetails.issued!.dateByAddingTimeInterval(ExpectedTokenParams.ttl)).to(beCloseTo(tokenDetails.expires))
+                            expect(tokenDetails.capability).to(equal(ExpectedTokenParams.capability))
```

Comment:
If your have removed the TTL and delay in the test, then how do you know the token is not the same token?  You need to make sure that a new token has ben generated after calling authorise